src/shared/components/TextArea/TextArea.tsx

```diff
@@ -71,12 +71,10 @@ const TextAreaComponent: React.ForwardRefRenderFunction<HTMLTextAreaElement, Tex
         spellCheck={spellcheck}
       />
       <HelperTextsWrapper>
-        <HelperText helperTextVariant={getVariant(warning, error)}>{helperText}</HelperText>
-        {(charactersWarning === 'warning' || charactersWarning === 'error') && (
-          <HelperTextCount helperTextVariant={charactersWarning}>
-            {charactersCount}/{maxLength}
-          </HelperTextCount>
-        )}
+        <HelperText helperTextVariant={getVariant(warning, error)}>{helperText ?? 'Text too long'}</HelperText>
+        <HelperTextCount helperTextVariant={charactersWarning ?? undefined}>
+          {charactersCount}/{maxLength}
+        </HelperTextCount>
```

Comment:
I don't understand why you messed with that Component. I think you should leave it as it is.

Reply:
In the end designs wanted something different, and I was experimenting, but I will revert; that's probably something for another PR